homeworkJava8/src/main/java/homework06/CarMain.java

```diff
@@ -0,0 +1,82 @@
+package homework06;
+
+import java.util.ArrayList;
+import java.util.*;
```

Comment:
try getting used to not importing * (since you don't use all of the imports), just import what you need. Also, in your case you import everything (*) and then List and Collectors